diff --git a/.gitignore b/.gitignore
index d369ddf..6ff3ae2 100644
--- a/.gitignore
+++ b/.gitignore
@@ -21,9 +21,11 @@ core-v-mini-mcu.upf
 core-v-mini-mcu.dc.upf
 tb/tb_util.svh
 hw/core-v-mini-mcu/include/core_v_mini_mcu_pkg.sv
+hw/core-v-mini-mcu/core_v_mini_mcu.sv
 hw/core-v-mini-mcu/system_bus.sv
 hw/core-v-mini-mcu/system_xbar.sv
 hw/core-v-mini-mcu/memory_subsystem.sv
+hw/core-v-mini-mcu/peripheral_subsystem.sv
 hw/system/x_heep_system.sv
 hw/system/pad_ring.sv
 tb/tb_util.svh
